client/sites-dashboard/controller.tsx

```diff
@@ -24,6 +24,7 @@ export function sitesDashboard( context: PageJSContext, next: () => void ) {
 			<SitesDashboard
 				queryParams={ {
 					search: context.query.search,
+					showHidden: context.query[ 'show-hidden' ],
```

Comment:
At runtime the value of `showHidden` is going to end up being a string, even though the prop says its a `boolean`. Generally that's going to work out ok, but could lead to some surprising behaviour down the track where the TypeScript engine insists that something isn't possible, even though at rutime it is possible.

I mentioned above that I don't actually think we need the `show-hidden` query param, but if we are going to keep it maybe we should for this into a boolean and make sure `show-hidden=false` has the expected behaviour.
```suggestion
					showHidden: context.query[ 'show-hidden' ] === 'true',
```